packages/react/src/patterns/sub-patterns/LinkList/README.md

```diff
@@ -0,0 +1,87 @@
+# Link List
+
+> The Link List component will be used to have list of different cta types.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/link-list';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LinkList } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const items = [
+      {
+        heading: 'Containerization: A Complete Guide',
+        type: local,
+        copy: 'Lorem ipsum dolor sit amet',
+        href: 'https://ibm.com',
+      },
+      {
+        heading: 'Why should you use microservices and containers?',
+        type: external,
+        copy: 'Lorem ipsum dolor sit amet',
+        href: 'https://ibm.com',
+      },
```

Comment:
Looks like a syntax error here. It doesn't appear as if you're closing the array.